docs/UserGuide.md

```diff
@@ -117,6 +117,9 @@ Features relating to the display of information within these panels are describe
 * A candidate's `course` can only be either of the 5 computing courses in NUS School of Computing i.e. Business Analytics, Computer Engineering, Computer Science, Information Security or Information Systems.
   * Reason being, our **Target User Profile** is restricted to NUS SoC.
 
+* A candidate's `seniority` ranges as such [1, 4], i.e. 1, 2, 3 or 4. It will be displayed as `COMX` where X is [1, 4].
+  * To address seniority of students in double-degree programmes, it has been discussed that `COM5` has not been seen in NUS SoC system. Thus, the restriction of the range [1, 4].
```

Comment:
I'm wondering if we could perhaps adjust this phrasing to something like: `The seniority attribute field serves as a gauge for the SoC professor on which year the candidate is in to better understand how far along they are in their undergraduate studies. It follows the faculty classification from COM1 to COM4, and does not differentiate beyond COM4 following the faculty system classification.`

Because i don't think year 5 is solely for DDP students, but also for anybody who isn't able to graduate within the usually 4 years.

Reply:
have added it in. thanks for this! makes it clearer 👍🏻